### PR TITLE
{bio}[GCC/8.2.0-2.31.1] BBMap v38.87

### PR DESCRIPTION
**Pull request by agimenog**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn152.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/55b12996392f3077b92c49527db2aa4a for a full test report.

File: easybuild/easyconfigs/b/BBMap/BBMap-38.87-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['22ab642b8af88faf208a56763158da895004c5231df572d3163ce52fbfb63240']
+
+dependencies = [('Java', '11', '', True)]
```
Comment (casparvl) [inline easybuild/easyconfigs/b/BBMap/BBMap-38.87-GCC-8.2.0-2.31.1.eb:22]: While this works, the usage of `True` for indicating that this dependency is at the system toolchain level is now deprecated. Please replace with `dependencies = [('Java', '11', '', SYSTEM)]`.

**Review by casparvl — APPROVED** [review]
Lgtm!

File: easybuild/easyconfigs/b/BBMap/BBMap-38.87-GCC-8.2.0-2.31.1.eb (hunk 1 above)
Comment (agimenog) [reply easybuild/easyconfigs/b/BBMap/BBMap-38.87-GCC-8.2.0-2.31.1.eb]: done! please check @casparvl

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn480.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/cf13b30266afac432aea4ced4b38b01c for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @agimenog!

**Pull request merged**